docker-compose.example.yml

```diff
@@ -25,7 +25,7 @@ services:
       # Change This: The publicly exposed url for the api
       # For example if the server is available from the domain example.com,
       # change this value to http://example.com/api
-      - API_ENDPOINT=http://localhost:8080/api
+      - API_ENDPOINT=http://localhost:8080/
       - PUBLIC_ENDPOINT=http://localhost:8080/
       - API_LISTEN_PORT=80
```

Comment:
This could be removed as the standalone API has been removed

Reply:
I don't think it has.  These changes affect the production docker image, but the "standalone api" will still be used during development of the frontend when using the parcel dev server.

Reply:
I see, we can't remove that. We could make it default to port 80 if no environment variable is given, but that should be a pull request for itself.